src/scripts/components/NotificationHubIcon.js

```diff
@@ -1,15 +1,36 @@
 import { __ } from '@wordpress/i18n';
 import { useShortcut } from '@wordpress/keyboard-shortcuts';
+import classNames from 'classnames';
 
-export const NotificationHubIcon = ( { toggle, isActive } ) => {
+/**
+ * Notification icon UI component
+ *
+ * @param {Object}    Props
+ * @param {Function}  Props.toggle   Toggle the drawer on and off.
+ * @param {boolean}   Props.isActive Predicate of whether the drawer is in an active state.
+ * @param {string[]=} Props.classes  The icon class names (defaults to [ 'dashicons-bell' ])
+ * @return {JSX.Element} - the Notification icon
+ */
+export const NotificationHubIcon = ( {
+	toggle,
+	isActive,
+	classes = [ 'dashicons-bell' ],
+} ) => {
+	/**
+	 * Enables the shortcut to close the drawer with the escape key
+	 */
 	useShortcut( 'wp-feature-notifications/close-drawer', () => {
 		if ( isActive ) toggle();
 	} );
 
 	return (
-		<button className={ 'ab-item' } aria-haspopup="menu" onClick={ toggle }>
+		<button
+			className={ 'ab-item' }
```

Comment:
Is this a WordPress class name, its is very terse.

Reply:
Yes, I have to use it to inherit the style from navbar icon!